packages/components/src/calendar/date-picker.js

```diff
@@ -127,6 +127,8 @@ class DatePicker extends Component {
 									this.onDateChange,
 									onToggle
 								) }
+								// onMonthPreviewed is required to prevent a React error from happening.
+								onMonthPreviewed={ noop }
```

Comment:
`onMonthPreviewed` is a required prop, as you can see by it's use in DateTimePicker, not sure why it doesn't default to `noop` in it's own component:
https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/date-time/index.js#L29